test/mocha.spec.js

```diff
@@ -32,4 +32,133 @@ describe('Mocha', function () {
       });
     });
   });
+
+  describe('.addFile()', function () {
+    it('should add the given file to the files array', function () {
+      var mocha = new Mocha(blankOpts);
+      mocha.addFile('myFile.js');
+      mocha.files.length.should.equal(1);
+      mocha.files[0].should.equal('myFile.js');
+    });
+  });
+
+  describe('.invert()', function () {
+    it('should set the invert option to true', function () {
+      var mocha = new Mocha(blankOpts);
+      mocha.invert();
+      mocha.options.invert.should.equal(true);
+    });
+
+    it('should be chainable', function () {
+      var mocha = new Mocha(blankOpts);
+      mocha.invert().should.equal(mocha);
+    });
+  });
+
+  describe('.ignoreLeaks()', function () {
+    it('should set the ignoreLeaks option to true when param equals true', function () {
+      var mocha = new Mocha(blankOpts);
+      mocha.ignoreLeaks(true);
+      mocha.options.ignoreLeaks.should.equal(true);
+    });
+
+    it('should set the ignoreLeaks option to true when param equals false', function () {
+      var mocha = new Mocha(blankOpts);
+      mocha.ignoreLeaks(false);
+      mocha.options.ignoreLeaks.should.equal(false);
+    });
+
+    it('should be chainable', function () {
+      var mocha = new Mocha(blankOpts);
+      mocha.ignoreLeaks(false).should.equal(mocha);
+    });
+  });
```

Comment:
Do we want to also test the behavior of `.ignoreLeaks()` (no parameter) while we're at it?